vnext/ReactUWP/Base/CoreUIManagers.cpp

```diff
@@ -6,7 +6,7 @@
 #include <Modules/NativeUIManager.h>
 #include <ReactUWP/IReactInstance.h>
 #include <ReactUWP/ViewManagerProvider.h>
-#include <ReactWindowsCore/IUIManager.h>
+#include <Shared/IUIManager.h>
```

Comment:
is the Shared/ prefix here and elsewhere, needed? If Shared is in the include path this should not be necessary

Reply:
I'll give it a try.